templates/includes/sidebar.html

```diff
@@ -1,4 +1,13 @@
 <ul class="sidebar-nav">
+  {% if user.is_authenticated %}
+    <li>User: <a href="{% url 'taxi:driver-detail' user.id %}">{{ user.username }}</a></li>
+    <li><a href="{% url 'logout' %}">Logout</a></li>
+    <br>
+  {% else %}
+    <li>
+      <a href="{% url 'login' %}"></a>
```

Comment:
The login link is missing text. Consider adding descriptive text inside the `<a>` tag, such as 'Login', to make it clear to users what the link is for.